public/WEB-INF/web.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Does all this JSP web app stuff need to be included here?

Reply:
Does it need to be, no.  I didn't realize that I had left it in there.  It does not hurt anything and is something that we need for IMH pilot as they will host on tomcat so I think leaving it in is ok even though it could have been a task to itself.